### PR TITLE
azure - fix azure output storage logging

### PR DESCRIPTION
**Pull request by aluong**
- Adding minutes and seconds support for storage logs
- fixed the directory being used for outputting to storage account

### CODE REVIEW

File: tools/c7n_azure/c7n_azure/output.py [hunk 1]
```diff
@@ -43,16 +43,17 @@ class AzureStorageOutput(DirectoryOutput):
...
         super(AzureStorageOutput, self).__init__(ctx, config)
         self.log = logging.getLogger('custodian.output')
-        self.date_path = datetime.datetime.now().strftime('%Y/%m/%d/%H')
+
+        # folder structure Year/Month/Day/Hour/Second
+        self.date_path = datetime.datetime.now().strftime('%Y/%m/%d/%H/%M/%S')
         self.root_dir = tempfile.mkdtemp()
         self.blob_service, self.container, self.file_prefix = \
-            self.get_blob_client_wrapper(self.root_dir, ctx)
+            self.get_blob_client_wrapper(self.ctx.options.output_dir, ctx)
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/output.py]: oops, that was me.

File: tools/c7n_azure/c7n_azure/output.py [hunk 2]
```diff
@@ -43,16 +43,17 @@ class AzureStorageOutput(DirectoryOutput):
     def __init__(self, ctx, config=None):
         super(AzureStorageOutput, self).__init__(ctx, config)
         self.log = logging.getLogger('custodian.output')
-        self.date_path = datetime.datetime.now().strftime('%Y/%m/%d/%H')
+
+        # folder structure Year/Month/Day/Hour/Second
+        self.date_path = datetime.datetime.now().strftime('%Y/%m/%d/%H/%M/%S')
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/output.py]: fwiw, `-s` on the cli now supports arbitrary date formatting via pyformat (`now` is the datetime variable for interpolation).

Comment (aluong) [reply tools/c7n_azure/c7n_azure/output.py]: Is this documented? I couldn't find an example

Comment (kapilt) [reply tools/c7n_azure/c7n_azure/output.py]: not really.. its fairly recent as part of the output refactor

Comment (aluong) [reply tools/c7n_azure/c7n_azure/output.py]: I added an example to our docs

**Pull request merged**